src/Components/ResultSearchedCard/__test__/ResultSearchedCard.test.jsx

```diff
@@ -0,0 +1,85 @@
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import App from "../../../App";
+import { cities } from "citiesToFetchData.js";
+const nock = require("nock");
+
+window.alert = jest.fn();
+jest.mock("citiesToFetchData.js", () => ({
+	cities: ["Paris"],
+}));
+
+describe("Search", () => {
+	it("should show ResultSeatchedCard when you find your city and click search button", async () => {
+		window.alert.mockClear();
+
+		const WeatherCardList = nock("http://api.weatherapi.com")
+			.defaultReplyHeaders({
+				"access-control-allow-origin": "*",
+			})
+			.get("/v1/current.json?key=c6068278a47d4030aad124640241101&q=Paris&aqi=no")
+			.reply(200, {
+				current: {
+					condition: {
+						text: "Sunny",
+						icon: "icon",
+					},
+					temp_c: 10,
+					wind_kph: 10,
+				},
+				location: {
+					country: "France",
+					name: "Paris",
+				},
+			})
+			.get("/v1/current.json?key=c6068278a47d4030aad124640241101&q=Gujranwala&aqi=no")
+			.reply(200, {
+				current: {
+					condition: {
+						text: "Sunny",
+						icon: "icon",
+					},
+					temp_c: 10,
+					wind_kph: 10,
+				},
+				location: {
+					country: "Pakistan ",
+					name: "Gujranwala",
+				},
+			});
```

Comment:
mocking takes more lines than an actual testing part. How about creating a test utils that would allow you to re-use these mocks?